src/screens/Feed/components/PostImage/component.js

```diff
@@ -66,14 +65,14 @@ export class PostImage extends PureComponent {
   mounted = false
 
   render() {
-    const { uri, borderRadius, overlayColor } = this.props;
+    const { uri, borderRadius } = this.props;
     const { width, height } = this.state;
 
     return (
-      <Image
-        resizeMode="cover"
+      <FastImage
+        resizeMode={FastImage.resizeMode.cover}
```

Comment:
change to `cover` instead of `FastImage.resizeMode.cover` to keep with consistency